Sources/KountDataCollector/BTKountDataCollector.m

```diff
@@ -0,0 +1,7 @@
+#if __has_include(<Braintree/KountDataCollector.h>)
+#import <Braintree/KountDataCollector.h>
+#else
+#import <KountDataCollector/KountDataCollector.h>
+#endif
+
+int _keep;
```

Comment:
For SPM we need a dummy `.m` file as we cannot have a module with only `.h` files ([reference 1](https://forums.swift.org/t/swift-package-and-xcframework-target-for-c-library-where-to-include-the-header/51163) and [reference 2](https://stackoverflow.com/questions/68502255/how-to-define-obj-c-target-with-a-header-file-so-other-targets-can-implement-it)). 

There are a few ways to go about this we can either have something like we are using here `int _keep` or we could likely do something like `void __dummy() {}`. Essentially the file needs to have something other than an import statement in it. Was thinking we should document this somehow - maybe with just a note describing this and the 2 links from above? I know we don't typically add doc strings for internal things, but this seems like a good thing to document. Thoughts?

Reply:
+1 on a doc string here for internal explanation